settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "FastAPI City Temperature"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

Comment:
The type hint `str | None` uses the new union operator introduced in Python 3.10. If you need to support older Python versions, consider using `Optional[str]` from the `typing` module.